frontend/src/store.js

```diff
@@ -51,14 +57,23 @@ export default new Vuex.Store({
       } else {
         newAppCfg.serviceClients = [{
           accountName: newAppCfg.applicationAcronym + "_SERVICE_CLIENT",
+          secret: "",
           oauthScopes: [],
           oauthGrantTypes: [],
           oauthRedirectUrls: [],
           oauthAccessTokenValidity: null,
           oauthRefreshTokenValidity: null,
           oauthAdditionalInformation: "{\"autoapprove\":\"true\"}",
           authorizations: []
-        }]
+        }];
+
+        if (state.userAppCfg.deploymentMethod === "deploymentManual") {
+          newAppCfg.serviceClients[0].secret = "${" + newAppCfg.serviceClients[0].accountName + ".password}";
```

Comment:
Perhaps ES6 templating like the following would be good:
`\$\{ ${newAppCfg.serviceClients[0].accountName}.password\}`